website/src/components/SequenceDetailsPage/SequenceDataUI.tsx

```diff
@@ -96,6 +98,7 @@ export const SequenceDataUI: FC<Props> = ({
                         />
                     )}
 
+                    {/* TODO ! */}
```

Comment:
👀

Reply:
Thanks, this was actually already done :)